resources/cms/values.yaml

```diff
@@ -1,10 +1,10 @@
 global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
-  cmsControllerManager:
-    dir: develop/
-    version: '5bad2085'
+  cms_controller_manager:
```

Comment:
Please don't change name! https://github.com/kyma-project/kyma/blob/release-1.4/resources/cms/charts/cms-controller-manager/templates/statefulset.yaml#L28

Reply:
Good catch! Thank you